conf/map_darkstar.conf

```diff
@@ -171,7 +171,13 @@ max_merit_points: 30
 #Minimum time between uses of yell command (in seconds).
 yell_cooldown: 30
 
-#Audit[logging] settings
+#Command Audit[logging] commands with lower permission than this will not be logged.
+# Zero for no logging at all. Commands given to non GMs are not logged. Autotranslate is not parsed.
+audit_gm_cmd: 0
+

```

Comment:
What if I wanted to create a command with permission level 0? How would I audit that?

Reply:
you don't. permission zero is intentionally ignored - don't give commands to everyone in creation if you can't trust that command to be safe :)

Reply:
1 = log everything no matter how high the tier. we just don't log zero because if you were dumb enough to give something dangerous to tier zero you are beyond help :)